roseus_bt/README.md

```diff
@@ -0,0 +1,50 @@
+roseus_bt
+============
```

Comment:
```suggestion
# roseus_bt
```

Reply:
both roseus_smach and roseus_mongo seems to have the bar syntax. Any reason to change to the hashtag title?

Reply:
If you want to use `====`, you should format it to the same lenght with `roseus_bt`.
however, in other part, you use `#` format, so it is better to use `#` format for all parts.
`==` and `--` are official formats but alternative for `#`, and we should not mix them.

https://www.markdownguide.org/basic-syntax/

Reply:
Oh, didn't realize they were actually producing the same styling. Thanks for the heads up!